infra-templates/ipsec/16/rancher-compose.yml

```diff
@@ -21,6 +21,18 @@
       type: "string"
       default: '10.42.0.0/16'
       required: true
+    - variable: "SUBNET_START_ADDRESS"
+      label: "Subnet start IP Address"
+      description: "Proceed with caution; typical users will not need to change these. Incorrect values can break your Rancher installation."
```

Comment:
Let's reword to: `This is an advanced setting and most users will not need to use this. Incorrect values can break your Rancher installation.`